contracts/utils/Address.sol

```diff
@@ -138,10 +166,7 @@ library Address {
     function _revert(bytes memory returndata) private pure {
```

Comment:
In the current state, this private `_revert(..)` function can be removed/merged in the calling function.

Reply:
this function is called 2 times. Are you saying we should manually inline/duplicate ?